scripts/cozy-app-dev.sh

```diff
@@ -69,7 +69,7 @@ do_start() {
 		echo "ok"
 	fi
 
-	trap "trap - SIGTERM && kill 2>&1 > /dev/null -- -${$}" SIGINT SIGTERM EXIT
+	trap 'trap - SIGTERM && kill 2>&1 > /dev/null -- -${$}' SIGINT SIGTERM EXIT
```

Comment:
I don't understand this line. So maybe my question is stupid, but changing the quotes doesn't change the interpolation of `${$}`?